### DISCUSSION TITLE
Photoirradiation for reduction of endogenous Autofluorescence

### DISCUSSION THREAD

**Discussion by CroteJ-BL** [IBEXImagingCommunity/ibex_imaging_knowledge_base#44] [Q&A]
I'm looking for advice regarding construction of a "lightbox" of some sort to photo irradiate tissue to reduce autofluorescence in certain human tissues (FFPE). We've found a couple citations that reference older LED array products, but, unfortunately, it seems that most of these products are discontinued, or, the manufacturers are no longer in business. Has anyone had success in this area?

**Comment by DippedPretzels** [comment]
For bleaching with LEDs, you may have seen these previous papers:

https://www.sciencedirect.com/science/article/pii/S0165027020303472
and:
https://www.sciencedirect.com/science/article/pii/S0165027013002896?via%3Dihub

If you want to bleach a very large batch, then industrial white LED lighting will get you the highest lumens for the lowest cost.  These are things like "high bay UFO lighting" and you won't find anything with more lumens for a lower price:
[](https://www.1000bulbs.com/product/220386/PLT-11908.html)
 
These are arrays of blue LED emitters with a phosphor to yield spectra like this:
![image](https://user-images.githubusercontent.com/26780277/227370009-71ca15bd-1091-48a7-8ca2-19d2fe9abd8a.png)


For white LEDs, note the color temperature - the above is 5000K.  If you find red fluorophores are not getting bleached, something like 3000K may be better:
![image](https://user-images.githubusercontent.com/26780277/227370113-e054978a-afcd-489b-adfd-d04cd2bb2dea.png)


However most will drop sharply past 700nm.  If you need to bleach redder than that, you'll have to get additional LEDs of different wavelengths.  There are exotic industrial LEDs that push into 1000nm https://www.effilux.com/en/products/hyperspectral
 
![image](https://user-images.githubusercontent.com/26780277/227370318-402f70b1-abb6-4551-b83c-f33777e93a8f.png)



But these are likely very expensive.

 

The problem with high bay LED lighting is that unless you're doing a very large batch, the LED chips are spread across a large area, so most light will be wasted.  For one or few slides, you're better off with a single chip LED.  White plant grow lights are fine for this:

[](https://www.amazon.com/Grow-Spectrum-Dimmable-MeanWell-Flowering/dp/B06XFNXLBK/ref=sr_1_3?crid=1PB7URUX7YTUJ&keywords=cree%2BLED%2BCOB%2Bcxb3590&qid=1658768022&sprefix=cree%2Bled%2Bcob%2Bcxb3590%2Caps%2C60&sr=8-3&th=1)
This comes with a CREE CXB3590 COB LED chip - always stay away from the cheapest LEDs you find, as there are a lot of counterfeit and poor quality "100W" chips online.

 

Note the light spill on these single chip LEDs will also be large - if needed, you can remove the existing optic on the chip and replace it with a large TIR collimator:

https://www.ledwerx.com/llc56n-collimator-in-pmma-for-cobs-and-high-power-leds/

However, you'll need to find a way to mount it.

 

If you are looking to bleach a specific wavelength range, or supplement a white LED, you should look at 100W chips on this website:
https://www.leds-global.com/
But you'll need to assemble it yourself and buy a 100W LED driver and heatsink.  That site also sells UV and IR LEDS. 

 

Stay far away from any LED chips that would instruct you to solder 120V AC wires directly to the chip - these look like fire hazards to me.

 

 

 

With any light source you pick, your largest concern will likely be managing heat generated on your sample - I know one group that bleaches inside a walk-in fridge.  You could try keeping the slides in a very large PBS bath or you could mount the light and slides at an angle, and flow PBS across the slides into a waste bucket.  For the first bleaching, you should monitor temperatures on the slide with a non-contact thermometer, and back the light source away until it reaches an acceptable steady state temperature.

 

Hope this helps!
-Spencer

**Reply by zmikulski** [reply]
Hi All, 

We had success with 15000 lux light therapy LED panels from Amazon. They are small but you can fit 2 trays (with a multi-well footprint, they sit a bit tilted but it's OK) and bleach 8 slides at a time. We also experimented with vertical bleaching, where slides are placed in racks, and while not as efficient as the horizontal way it still works reasonably well. 

It is important to seal the edges of the panel with some silicone or waterproof it in another way. We flooded, opened them up to dry, and then flooded them again and fried one of our lights because we didn't do that. That allowed us to take a good peek inside - they are decently built, with a large diffuser panel in the middle and LED strips that are fused to the aluminum supports for heat dissipation. I was worried about overheating but the slides in the bleaching bath did not warm up too much (~40C if I remember correctly, but it might be higher). 

We sandwich our slides in reservoirs and bleach them with 2 panels. These lights are very bright. Laser safety goggles will keep comfortable and safe while working with them. Speaking of safety, did I already mention the electrical shock hazard if you flood them? Be careful :)

There are a couple of listings on Amazon that claim to be 15k lux but they don't always deliver that. 
We tested a couple of lights with the 'Physics Toolbox app' for Android by Vieyra Software (an amazing toolbox by the way!!!). It looks like the ones in the listings: 
https://www.amazon.com/Sunlight-Temperature-Adjustable-Brightness-Function/dp/B09XQPC5FG/
https://www.amazon.com/DIKAKO-Temperature-Adjustable-Brightness-Rotatable/dp/B09WVNDTLC
https://www.amazon.com/Temperature-Adjustable-Brightness-Sunlight-Foldable/dp/B0BMKVNND9

and delivers 15k as promised. They are between $20-40, which I think is a crazy value for the punch they deliver. Unfortunately, the stock is often low so if you grab a pair that you like, I'd recommend buying at least one more to have a spare. 

Hope it will be helpful and please reach out if you have any questions. 
Best, Z.

**Comment by radtkea** [comment]
Hi Z!!!

Wow, this is amazingly helpful! Thank you for your very detailed and thought-filled response! I greatly appreciate the safety tips! Very important.

Apologies to getting back to you so late. I missed/didn't receive updates. I will share with the others.

Thank you for offering your support and guidance.

Have a lovely day and best of luck with your experiments! Excited to see your results.
A
